src/Header/Header.js

```diff
@@ -1,19 +1,16 @@
 import React from 'react'
+import { Link } from 'react-router-dom'
 import PropTypes from 'prop-types'
 import clapboard from '../assets/clapboard.png'
 import tomatillo from '../assets/tomatillo.png'
 import '../scss/_Header.scss'
 
-const Header = ({ loginStatus, showLoginPage, logOut, name }) => {
+const Header = ({ loginStatus, logOut, name }) => {
   return (
     <header>
       <div className="title">
-        <img
-          className="logo"
-          src={tomatillo}
-          alt="Website Icon"
-        />
-        <h1>Rancid Tomatillos</h1>
+        <img className="logo" src={tomatillo} alt="Website Icon" />
+        <Link to="/"><h1>Rancid Tomatillos</h1></Link>
```

Comment:
LOVE THIS! A lot of web pages use the title of the page as a way to go back to the home page. Great idea!